src/zenml/integrations/label_studio/__init__.py

```diff
@@ -26,7 +26,7 @@ class LabelStudioIntegration(Integration):
     """Definition of Label Studio integration for ZenML."""
 
     NAME = LABEL_STUDIO
-    REQUIREMENTS = ["label-studio==1.6.0", "label-studio-sdk==0.0.15"]
+    REQUIREMENTS = ["label-studio==1.6.0", "label-studio-sdk>=0.0.17"]
```

Comment:
Would we want to limit this to the current version instead of fully open range?

Reply:
In general we're trying to open up the ranges of the versions we support for all integrations, so the minimum version to be installed is 0.0.17. Label Studio is one of our integrations with the most dependency conflicts, so I thought it's probably better to open it up?

Reply:
Ah yes, i guess just brought it up because just had an issue with this in mlflow